src/hb-ot-cmap-table.hh

```diff
@@ -521,21 +521,22 @@ struct cmap
 			       hb_prealloced_array_t<CmapSubtableLongGroup> *groups) const
   {
     CmapSubtableLongGroup *group = nullptr;
-    for (unsigned int i = 0; i < plan->codepoints.len; i++) {
-
-      hb_codepoint_t cp = plan->codepoints[i];
-      if (!group || cp - 1 != group->endCharCode)
+    hb_codepoint_t cp = HB_SET_VALUE_INVALID;
+    while (hb_set_next (plan->codepoints, &cp))
```

Comment:
I actually like the for() loop of this instead of while. But feel free to leave as is.